examples/employee-app/src/main/java/com/example/helidon/employee/ReposFactory.java

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Oracle and/or its affiliates. All rights reserved. */
+package com.example.helidon.employee;
+
+public class ReposFactory {
+
+    public static EmployeeRepository getRepo(String driverType) {
+        switch (driverType) {
+            case "Array":
+                return new EmployeeRepositoryImpl();
+            case "Oracle":
+                return new EmployeeRepositoryImplDB();
+            default:
+                System.out.println("No database driver found...");
+                System.exit(1);

```

Comment:
This is wrong. Consider move this to the main class instead.

Reply:
Doh! Don't know what I was thinking at the time. Should be fixed now.